src/renderer/viz/expressions/placement.js

```diff
@@ -7,14 +7,6 @@ import { checkLooseType, checkType, implicitCast } from './utils';
  * - `symbolPlacement: placement(0,0)` means to align the center of the image with the point center.
  * - `symbolPlacement: placement(-1,-1)` means to align the top right corner of the image with the point center.
  *
- *           |1
- *           |
- *           |
- * -1 -------+------- 1
- *           |
- *           |
- *         -1|
- *
```

Comment:
Can we keep this? 🙏

Reply:
https://github.com/CartoDB/carto-vl/pull/855#discussion_r210871209